config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy@sha256:db06cc4c084dd0253134f156dddaaf53ef1c3fb3cc809e5d81711baa4029ea4c
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0
```

Comment:
I think we use the digest instead of tag here so that it can work in disconnected environments

Reply:
Fixed. Thanks!